### ISSUE TITLE
Cosima still unplayable

### ISSUE THREAD

**Issue by mattelonian** [magefree/mage#8379]
You can't play cosima as a bounce commander because the choice to move her into the command zone happens only once and remains true for the entire game, so either she's a useless commander or you have to play it so she can never return to the command zone. This isn't in accordance to how it works under actual commander rules, since you should be able to choose each time whether or not your commander returns to the command zone whenever it changes zones.

**Comment by Alex-Vasile** [comment]
Duplicate of #8126
